### ISSUE TITLE
Add Ownbit Wallet

### ISSUE THREAD

**Issue by jtomtan** [ethereum/ethereum-org-website#12528]
### Wallet name

Ownbit

### Wallet description

1. Ownbit is a Cold wallet and MultiSig wallet for popular crypto assets, like Bitcoin, Ethereum, TRON, Solana and so on. Ownbit implements Cold wallet features by using two mobile phones (no specific hardware needed). One mobile offline as the cold wallet, and the other online as the Watch wallet. It is as secure as hardware wallets.

### Wallet logo

![ownbit_logo](https://github.com/ethereum/ethereum-org-website/assets/11899268/4914518a-6256-4a92-a202-26d320872d82)

### Background color for brand logo

#FFFFFF

### URL to the project

https://ownbit.io/

### When did the wallet go live to users?

2018

### Does the wallet have an active development team?

Yes. Github: https://github.com/bitbill

### Is the wallet globally accessible?

Globally accessible. 

### Is the wallet available in multiple languages?

English / Chinese Simplified / Traditional Chinese / Russian / Japanese / Korean / German / Spanish

### What social links are there for the project?

https://twitter.com/OwnbitWallet

### Does the wallet have a mobile app? If yes, which operating systems are supported (iOS, Android)?

iOS: https://apps.apple.com/us/app/ownbit-cold-multisig-wallet/id1321798216
Android: https://play.google.com/store/apps/details?id=com.bitbill.www

### Does the wallet have a desktop app? If yes, which operating systems are supported (Windows, Mac, Linux)?

No.

### Does the wallet have a browser extension? If yes, which browsers are supported (Chromium, Firefox, Safari)?

No.

### Is it a hardware wallet?

Yes. But it implements hardware wallet features in a different way. Ownbit uses two mobile phones to implement hardware wallet features. One mobile is permanently offline as the cold wallet. The other is online as the Watch wallet. They broadcast signed transactions by scanning QR codes. 

### Is the source code for the wallet fully open-source?

No. Open-source only partially, Github: https://github.com/bitbill

### What license is the wallet software released under?

MIT

### Who holds the private keys?

Users hold the private keys (mnemonic phrase). Assets' accessibility doesn't depend on Ownbit team at all. Even Ownbit team goes disappear tomorrow, all assets (including MultiSig assets) can be recovered easily. For non-MultiSig assets, users can just recover them in other BIP39/BIP44 compatible wallets by importing private keys or mnemonic phrase. MultiSig assets can also be recovered by calling specific Smart Contract functions.  'Assets self-recover help page' is accessible in the Ownbit app.

### Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible

For users who use Ownbit MultiSig features. Ownbit MultSig source code is open-sourced at: https://github.com/bitbill/ownbit-multisig-contracts. It has been used for more than 5 years, securelly stored dozens of millions USD assets and proved to be secure. The source code is not audited by a specific organization, but it is widely read and checked by other developers. And this Ownbit MultiSig source code is also used by some other projects and it has 40 stars now in the Github. 

And for Ownbit Cold wallet features, we have a way to prove it's absolute security. All users can check and approve it by their own. We have documented a page to welcome all our users to check and approve Ownbit's security: https://bitbill.oss-accelerate.aliyuncs.com/h5/app/ownbit_cold_wallet_is_more_secure_than_hardware_cold_wallet_en.html

### Has the wallet's smart contract code or security modules been audited?

As described above, not audited by specific organization, Github: https://github.com/bitbill/ownbit-multisig-contracts

### Does the wallet have an internal security team?

No special internal security team. It's a small team, but all members are very experienced, from 2013 as Bitcoin wallet developers. 

### Any other security testing that should be noted?

We are mainly focus on the security which can be checked from the User end. That's the say we want our users themselves to check Ownbit's security by logic. And we do have some users who have program background checked Ownbit's design as we mentioned in the page: https://bitbill.oss-accelerate.aliyuncs.com/h5/app/ownbit_cold_wallet_is_more_secure_than_hardware_cold_wallet_en.html
/

### Scam protection?

Yes. Ownbit focused on security. And when users use Ownbit to interact with Dapp, we do have some logic to warn users to be very careful. The app does some basic checking for potential scams.

### Does the wallet support connecting to a hardware wallet?

No

### Does the wallet support WalletConnect?

Yes

### Does the wallet support importing Ethereum RPC endpoints?

No.

### Does the wallet support viewing and interacting with NFTs?

Yes

### Does the wallet support connecting to Ethereum applications?

Yes. Through WalletConnect.

### Does the wallet support staking directly?

No.

### Does the wallet support swaps directly?

No.

### Does the wallet support multi-chain networks?

Yes. Support multi-chain is built-in. Currently supports popular multi-chain, like BNB smart chain, Matic, Arbitrum, Optimism, Zksync Era, etc.

### Does the wallet support Ethereum layer 2 networks?

Yes. Support is built-in. Like Arbitrum, Optimism, Zksync Era, etc.

### Does the wallet allow the user to customize gas fees?

Yes. In the send process, the last step, advanced options, users can customize gas fees, gasLimit and custom HEX data.

### Does the wallet support sending transactions to ENS addresses?

Yes, ENS is supported.

### Does the wallet support importing or automatically querying and displaying ERC-20 tokens?

Yes. Users can search and add ERC20 tokens by symbol, name or contract address.

### Does the wallet support EIP-1559 (type 2) transactions?

No. Currently still legacy. But have plan to upgrade to type 2.

### Does the wallet have fiat on-ramps through credit/debit cards, wire transfers, or bank transfers (ACH)?

No.

### Does the wallet support withdrawals to fiat?

No.

### Is the wallet a multi-signature wallet?

Yes. Users can check this page for how to use Ethereum MultiSig in Ownbit: https://ownbit.io/h5/app/prompt/multisig_howto_en.html

### Does the wallet support social recovery?

No.

### Who can the ethereum.org team can contact regarding the wallet in future?

You can contact Ownbit team through the following ways:

1. Email: support@ownbit.io
2. Telegram: https://t.me/ownbit
3. Twitter:  https://twitter.com/OwnbitWallet

### Does the wallet have a dedicated support team?

Ownbit has 7x24 support. Also can be accessed by: 

1. Email: support@ownbit.io
2. Telegram: https://t.me/ownbit

### What educational resources/documentation do you provide to users?

Documents or help information is built in the App. Can be accessed in App's homepage, top left corner, help icon.

### Does the wallet have any integrated tools not mentioned above?

Yes. Beside MultiSig feature, Ownbit also supports two types of accounts: external account (controlled by private keys) and smart contract account. That's to say, users can also create smart contract account in Ownbit, and implements features like assets locking, asset inheritance, etc.

### Would you like to work on this issue?

- [X] Yes
- [ ] No

**Comment by wackerow** [comment]
Hey @jtomtan, thanks for the request! Will assign this to @corwintines for a product review, but just took a quick read through and will kick off a couple questions:

"Actively maintained"... took a quick look at the repo and I don't see too much activity (only one update to any of the repos in the last year?)

<img width="681" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/d20ea01b-fbf6-4f5c-b5c5-f2f4c82a4423">

I can imagine part of this is a product of being a fairly mature codebase, but just wanted to note this to get your thoughts. From our side, we similarly care a lot about security and want to make sure users are being directed to products that are _actively_ being maintained, to handle any bugs or "supply-chain" issues. 

---

"Yes" to hardware wallet, but you mention you use phones. Am I correct that there is no separate dedicated device here? If not, I would probably just list this as a mobile wallet in the context of ethereum.org.

---

> No. Currently still legacy. But have plan to upgrade to type 2.

Looking forward to it... your users will thank you! =)

---

This otherwise looks good at a glance... will let @corwintines do a review and then I see you requested to work on this, so if all looks good will assign you when ready. Thanks again!

**Comment by jtomtan** [comment]
> Hey @jtomtan, thanks for the request! Will assign this to @corwintines for a product review, but just took a quick read through and will kick off a couple questions:
> 
> "Actively maintained"... took a quick look at the repo and I don't see too much activity (only one update to any of the repos in the last year?)
> 
> <img alt="image" width="681" src="https://private-user-images.githubusercontent.com/54227730/315588427-d20ea01b-fbf6-4f5c-b5c5-f2f4c82a4423.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTEwNjc2NzIsIm5iZiI6MTcxMTA2NzM3MiwicGF0aCI6Ii81NDIyNzczMC8zMTU1ODg0MjctZDIwZWEwMWItZmJmNi00ZjVjLWI1YzUtZjJmNGM4MmE0NDIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzIyVDAwMjkzMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1OWNkNWI4ZGY1ODU3NGQwNmNmM2U0NDg0ODMxOTU2N2FlZTg4MDFkZjM5ZjlmYzg4ZjY1YWM5NzIwNDcwOTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mV0ZG9ogDm_y934KL8qWYeyZ-E7ys1qZpLoBraX06vc">
> I can imagine part of this is a product of being a fairly mature codebase, but just wanted to note this to get your thoughts. From our side, we similarly care a lot about security and want to make sure users are being directed to products that are _actively_ being maintained, to handle any bugs or "supply-chain" issues.
> 
> "Yes" to hardware wallet, but you mention you use phones. Am I correct that there is no separate dedicated device here? If not, I would probably just list this as a mobile wallet in the context of ethereum.org.
> 
> > No. Currently still legacy. But have plan to upgrade to type 2.
> 
> Looking forward to it... your users will thank you! =)
> 
> This otherwise looks good at a glance... will let @corwintines do a review and then I see you requested to work on this, so if all looks good will assign you when ready. Thanks again!

Hi wackerow, Thanks very much for your comments. Because Ownbit is not (fully) open source. The most important repositories are private. They are actually actively maintained. You can check my screenshots here: 
![ownbit-github-screenshot](https://github.com/ethereum/ethereum-org-website/assets/11899268/f180ca6a-a55e-44cd-b0d6-ac405a7dc09c) Or you can also check Ownbit in iOS Appstore or Google play, new versions normally released in two or three weeks. 

For 'hardware wallet', I agree with your comments. We implemented the same functions as hardware wallet with just two mobile phones. There's not dedicated hardwares. So it’s not a hardware wallet in the traditional sense. So 'hardware wallet' can be marked as NO for Ownbit.

For type 2, yes. We would like to.

**Comment by wackerow** [comment]
Great, thanks for the reply @jtomtan! Have assigned for review; bear with us on timing here since in general product listings are a lower priority for the team with a bunch of stuff going on in parallel. 🙏

**Comment by jtomtan** [comment]
> Great, thanks for the reply @jtomtan! Have assigned for review; bear with us on timing here since in general product listings are a lower priority for the team with a bunch of stuff going on in parallel. 🙏

Understood. Will be waiting with patience. Thanks.

**Comment by jtomtan** [comment]
hi, @corwintines any update on this? @wackerow

**Comment by wackerow** [comment]
<img width="970" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/de2515e7-8677-4d04-8775-4524a6dd17dd">

(https://play.google.com/store/apps/details?id=com.bitbill.www)

> Ownbit provides services in a membership subscription model. Members are anonymous and will not be required to collect any personal information.
> After the free trial period expires, if you do not pay the membership fee, you cannot continue to use Ownbit (Sending function is blocked until membership is renewed). However, you can recover your assets in other BIP39 compatible wallets at any time by using mnemonic phrase or private keys.

@jtomtan Can a user perform basic functionality without paying? Reviews in the app stores and this notice in the app itself suggest otherwise.

**Comment by jtomtan** [comment]
> Can a user perform basic functionality without paying? Reviews in the app stores and this notice in the app itself suggest otherwise.

Hi, @wackerow Yes, new users have a free trial period, after which they must pay the membership fee to use the wallet features. A completely free wallet will leave us without any financial support to develop the wallet. Ownbit and other wallets have a different philosophy, we do not encourage trading as we believe that you will lose money by trading frequently. So you won’t see any trading features on Ownbit. We encourage long-term holdings and provide solutions that we think are the safest (cold wallets, multi-signatures, etc.) to help users manage digital assets. Collecting membership fees is our established and only source of income. This charge is very clear, and all users can clearly know this news during the free trial period. In addition, we do not force users to pay. Any user who does not want to use Ownbit (do not want to pay) can easily restore assets in other wallets. We provide help information on how to self-recover assets and how to self-recover multi-signature assets.

**Comment by wackerow** [comment]
<img width="334" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/fb707b25-f6d7-4b0b-8d3a-1c3f1289de93">

Sure, I certainly understand that the team needs revenue. Apologies though, could you please help me understand what users are paying a recurring fee for here? Is there insurance, or fee subsidies? Or is it strictly for your interface? I'm not sure I understand why some of these would be on a price curve and recurring basis, like for more addresses or an N-of-M multi-sig with more signers, or generating more addresses.

**Comment by jtomtan** [comment]
> <img alt="image" width="334" src="https://private-user-images.githubusercontent.com/54227730/324128285-fb707b25-f6d7-4b0b-8d3a-1c3f1289de93.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1NzczMjgsIm5iZiI6MTcxMzU3NzAyOCwicGF0aCI6Ii81NDIyNzczMC8zMjQxMjgyODUtZmI3MDdiMjUtZjZkNy00YjBiLThkM2EtMWMzZjEyODlkZTkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIwVDAxMzcwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5NmZjNGRmMDQ4NzE5YzA0ZmJmNmI5ZTNiNzk0ZDVhZTVkMDViODZkY2Y5YzlmYTBkNjVlYjNmYTdlODJmZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HWhCoGRKzLetw5SNLCa16Lqsiq7UTcQscZo4gXli9Ns">
> Sure, I certainly understand that the team needs revenue. Apologies though, could you please help me understand what users are paying a recurring fee for here? Is there insurance, or fee subsidies? Or is it strictly for your interface? I'm not sure I understand why some of these would be on a price curve and recurring basis, like for more addresses or an N-of-M multi-sig with more signers, or generating more addresses.

The membership plans you see here is only to distiguish you are a team or a peronal user. For teams who often needs MultiSig like 3-5 needs higher membership level. This membrship fee includes no insurance, or fee subsidies. But we do have such plans. We were developing one Ethereum feature, Account abstraction (ERC-4337) in our next major release, high membership users can have their Contract Accounts for free, this is a kind of fee subsides.

And there's also some advanced features membership users can benifit, like Tron Energy free delegation. This could also be considered as a fee subsidies for users.

**Comment by jtomtan** [comment]
> > <img alt="image" width="334" src="https://private-user-images.githubusercontent.com/54227730/324128285-fb707b25-f6d7-4b0b-8d3a-1c3f1289de93.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM1NzczMjgsIm5iZiI6MTcxMzU3NzAyOCwicGF0aCI6Ii81NDIyNzczMC8zMjQxMjgyODUtZmI3MDdiMjUtZjZkNy00YjBiLThkM2EtMWMzZjEyODlkZTkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIwVDAxMzcwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5NmZjNGRmMDQ4NzE5YzA0ZmJmNmI5ZTNiNzk0ZDVhZTVkMDViODZkY2Y5YzlmYTBkNjVlYjNmYTdlODJmZDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HWhCoGRKzLetw5SNLCa16Lqsiq7UTcQscZo4gXli9Ns">
> > Sure, I certainly understand that the team needs revenue. Apologies though, could you please help me understand what users are paying a recurring fee for here? Is there insurance, or fee subsidies? Or is it strictly for your interface? I'm not sure I understand why some of these would be on a price curve and recurring basis, like for more addresses or an N-of-M multi-sig with more signers, or generating more addresses.
> 
> The membership plans you see here is only to distiguish you are a team or a peronal user. For teams who often needs MultiSig like 3-5 needs higher membership level. This membrship fee includes no insurance, or fee subsidies. But we do have such plans. We were developing one Ethereum feature, Account abstraction (ERC-4337) in our next major release, high membership users can have their Contract Accounts for free, this is a kind of fee subsides.
> 
> And there's also some advanced features membership users can benifit, like Tron Energy free delegation. This could also be considered as a fee subsidies for users.

hi @wackerow What do you think about this or any conclusion about Ownbit wallet?

**Comment by wackerow** [comment]
Appreciate the additional information @jtomtan. I don't think this is the kind of experience we should be providing for users and visitors of ethereum.org. I understand that users retain access to their seed phrase with Ownbit, but the trial period then paywall with recurring payment feels like a dark-pattern of getting people to pay ongoing fees just to access basic functionality that is generally available for free, such as sending or using your keys to sign any messages, or just holding. I'm failing to see what value is brought here over existing solutions that, from what I can tell, offer the same or more for significantly less expense, or free.

I do appreciate the inquiry and wish the project and your team the best of course, but I'm going to close this out. As an aside, curious if you've considered other models of revenue generation for the team that are more proportional to usage that would allow users to freely use the app, but perhaps pay small fees for certain convenience factors offered inside the wallet app (ie, swapping, minting, etc).

**Comment by jtomtan** [comment]
> Appreciate the additional information @jtomtan. I don't think this is the kind of experience we should be providing for users and visitors of ethereum.org. I understand that users retain access to their seed phrase with Ownbit, but the trial period then paywall with recurring payment feels like a dark-pattern of getting people to pay ongoing fees just to access basic functionality that is generally available for free, such as sending or using your keys to sign any messages, or just holding. I'm failing to see what value is brought here over existing solutions that, from what I can tell, offer the same or more for significantly less expense, or free.
> 
> I do appreciate the inquiry and wish the project and your team the best of course, but I'm going to close this out. As an aside, curious if you've considered other models of revenue generation for the team that are more proportional to usage that would allow users to freely use the app, but perhaps pay small fees for certain convenience factors offered inside the wallet app (ie, swapping, minting, etc).

hi @wackerow I understand your point. One suggestion, add a "Paid" tag for wallets like Ownbit rather than excluding us. I think ethereum.org should be a platform to contain all reasonable solutions. 

From a long term of view, all (or most) FREE wallets will come to non-free. The inclusion of more commercial wallets could be beneficial for Ethereum’s further growth.

**Comment by konopkja** [comment]
> > Appreciate the additional information @jtomtan. I don't think this is the kind of experience we should be providing for users and visitors of ethereum.org. I understand that users retain access to their seed phrase with Ownbit, but the trial period then paywall with recurring payment feels like a dark-pattern of getting people to pay ongoing fees just to access basic functionality that is generally available for free, such as sending or using your keys to sign any messages, or just holding. I'm failing to see what value is brought here over existing solutions that, from what I can tell, offer the same or more for significantly less expense, or free.
> > I do appreciate the inquiry and wish the project and your team the best of course, but I'm going to close this out. As an aside, curious if you've considered other models of revenue generation for the team that are more proportional to usage that would allow users to freely use the app, but perhaps pay small fees for certain convenience factors offered inside the wallet app (ie, swapping, minting, etc).
> 
> hi @wackerow I understand your point. One suggestion, add a "Paid" tag for wallets like Ownbit rather than excluding us. I think ethereum.org should be a platform to contain all reasonable solutions.
> 
> From a long term of view, all (or most) FREE wallets will come to non-free. The inclusion of more commercial wallets could be beneficial for Ethereum’s further growth.

Showing users fee structure of each product is indeed interesting feature that we have started considering.

**Comment by jtomtan** [comment]
> > > Appreciate the additional information @jtomtan. I don't think this is the kind of experience we should be providing for users and visitors of ethereum.org. I understand that users retain access to their seed phrase with Ownbit, but the trial period then paywall with recurring payment feels like a dark-pattern of getting people to pay ongoing fees just to access basic functionality that is generally available for free, such as sending or using your keys to sign any messages, or just holding. I'm failing to see what value is brought here over existing solutions that, from what I can tell, offer the same or more for significantly less expense, or free.
> > > I do appreciate the inquiry and wish the project and your team the best of course, but I'm going to close this out. As an aside, curious if you've considered other models of revenue generation for the team that are more proportional to usage that would allow users to freely use the app, but perhaps pay small fees for certain convenience factors offered inside the wallet app (ie, swapping, minting, etc).
> > 
> > 
> > hi @wackerow I understand your point. One suggestion, add a "Paid" tag for wallets like Ownbit rather than excluding us. I think ethereum.org should be a platform to contain all reasonable solutions.
> > From a long term of view, all (or most) FREE wallets will come to non-free. The inclusion of more commercial wallets could be beneficial for Ethereum’s further growth.
> 
> Showing users fee structure of each product is indeed interesting feature that we have started considering.

Sounds great. So please reconsider listing Ownbit wallet when this new policy takes effect. @wackerow
